index.html

```diff
@@ -5,16 +5,83 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/grid.css"> 
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
-
 <body>
+<div class="container">
+    <div class="row"> <!--header-->
+        <header>
+            <div class="col-6">
+            <h1 class="logo-title">ALLAN JOHNSON</h1>
```

Comment:
Tiny point, having all caps can cause issues with accessibility:
https://github.com/humanmade/hm-pattern-library/issues/75#issuecomment-235672885

Not a biggy, but you can have them in normal case here, `Allan Johnson`, and create a `.caps` class in the CSS with `text-transform: uppercase`.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform